node/modules/storageminer.go

```diff
@@ -724,6 +721,19 @@ func StorageAuth(ctx helpers.MetricsCtx, ca v0api.Common) (sectorstorage.Storage
 	return sectorstorage.StorageAuth(headers), nil
 }
 
+func StorageAuthWithURL(url string) func(ctx helpers.MetricsCtx, ca v0api.Common) (sectorstorage.StorageAuth, error) {
+	log.Infow("Setting auth token based on URL", "url", url)
```

Comment:
This might log API secrets, which we don't want as miners need to be able to share logs with not-so-trusted people (e.g. us)